### ISSUE TITLE
not match for afore-> future infinitive active for absum

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alpheios-core#501]
Whitaker is missing the information about the verb absum. we do not have a match because alpheios looks for afore as verb and not absum
Is there anything that can be done to fix this from our side?

**Comment by balmas** [comment]
The only solution for the moment would be to add copies of the inflection data for adsum under the afore as the lemma. Do you want to do that?

**Comment by monzug** [comment]
sure. would it be like this?

absum | abesse_abfui_abfuturus | afore |   | infinitive | future

add the following

afore | -_-_- | afore |   | infinitive | future

**Comment by balmas** [comment]
yes, but I think you have to duplicate all of the entries for absum in order to get the full tables.

**Comment by monzug** [comment]
can you please give me an example of the entries then if I have to duplicate the absum verb? not sure what to do

**Comment by balmas** [comment]
well, actually, what do you want the table to say? *Should* it just be this one table with that one row ?

**Comment by monzug** [comment]
to match Whitaker I can add the ending abfore and then a footnote for
afore. I need to find a table in latin grammars to see which one is more
commonly used. I don't like the idea to create a table just for one match

**Comment by balmas** [comment]
Ok. Honestly I think unless afore is a really common form, I would probably just let it be as it is right now move this issue to the morphsvc repo so that we can take care of it when we add the ability to override the morph data returned by Whitaker.

**Comment by monzug** [comment]
yes, let's move to morphsvc repo. Thanks!
